src/app/state/account/saga.ts

```diff
@@ -19,7 +19,20 @@ export function* fetchAccount(action: PayloadAction<string>) {
 
   yield* put(actions.setLoading(true))
   const nic = yield* call(getOasisNic)
-  const publicKey = yield* call(addressToPublicKey, address)
+  let publicKey: Uint8Array
+  try {
+    publicKey = yield* call(addressToPublicKey, address)
+  } catch (error) {
```

Comment:
swallows the error, I'm opposed to this

Reply:
It dispatches an `accountError` action, so I think it transforms the errors, not swallows it... or am I missing something here?  (I probably am.)

Reply:
This is what the user will see, when the address in the URL is wrong:

![image](https://user-images.githubusercontent.com/2093792/191647079-373e706f-ceba-4b08-a10e-2c4398073348.png)

Reply:
The downside of this approach is that we will hit APIs to fetch delegations, transactions, debonding info when we already know URL is wrong. UI is in a bit weird state too. I would ask PO/PM if this layout is Ok.

Reply:
Yes, I'm going to change that.